vpn-network/vpn-network-impl/build.gradle

```diff
@@ -33,6 +33,8 @@ dependencies {
 
     implementation Kotlin.stdlib.jdk7
     implementation "com.squareup.logcat:logcat:_"
+//    implementation "com.duckduckgo.mobile.android:ddg-android-netguard-release:1.0.0"
+    implementation "com.github.duckduckgo:netguard:jitpack-SNAPSHOT"
```

Comment:
This is to be changed to version number once https://github.com/duckduckgo/netguard/pull/12 is merged and we create the version tag